rpcd/etc/openstack_deploy/env.d/nova.yml

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
I'm not entirely clear where `group_names` is being defined, it only shows up as a when check in rpc-o. https://github.com/rcbops/rpc-openstack/search?utf8=%E2%9C%93&q=group_names